packages/language-server/src/completion/completions.ts

```diff
@@ -147,17 +149,41 @@ function getSuggestionForBlockAttribute(
   return suggestions
 }
 
-export function getSuggestionForFieldAttribute(
+
+export async function getSuggestionForNativeTypes(
+  foundBlock: Block,
+  wordsBeforePosition: string[],
+  document: TextDocument
+): Promise<CompletionList | undefined> {
+  let activeFeatureFlag = await declaredNativeTypes(document)
+  if (foundBlock.type !== 'model' || !activeFeatureFlag || wordsBeforePosition.length < 2) {
+    return undefined
+  }
+
+  let prismaType = wordsBeforePosition[1].replace('?', '').replace('[]', '')
+  let suggestions = await getNativeTypes(document, prismaType)
```

Comment:
Variables can be a `const`, they don't change.

Can `wordsBeforePosition[1]` be undefined? It will make `.replace` throw.

Reply:
Ah, it can be `wordsBeforePosition.length < 2` 👍